css/main.css

```diff
@@ -44,7 +44,17 @@ a { text-decoration:none;}
     opacity: 0;
     filter: alpha(opacity=0);
   }
-  
+  .page-title ul li i:first-child{
+    margin-left:20px;
+  }
+  .page-title ul li{
```

Comment:
Is not a good practice to use several tag in a row `.page-title ul li`  
It's a good practice to set a class for every li and define it here:
```
HTML: <li class="list-item-style">...
CSS:  .list-item-style{
    display:inline-block;
    padding-right:15px;
}
```